lib/lightning/attempts/attempt.ex

```diff
@@ -20,10 +20,12 @@ defmodule Lightning.Attempt do
 
   @final_states [
     :success,
-    :cancelled,
     :failed,
+    :crashed,
+    :cancelled,
     :killed,
-    :crashed
+    :exception,
+    :lost
```

Comment:
in this case, timeout is actually a `kill` because we know it took too long, we still have contact with the worker, and the worker shuts it down. lost is that we don't know what happened with the worker and it's now 20% longer than it should have timed out.

this is good fodder for a discussion next week at standup though :-)